.github/workflows/tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Suite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php: [ 5.5, 5.6, "7.0", 7.1, 7.2, 7.3, 7.4 ]
+                proto_c_ext: [ true, false ]
+
+        name: PHP ${{ matrix.php }} Unit Test
+        steps:
+            - uses: actions/checkout@v2
+            - uses: codecov/codecov-action@v1
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: ${{ matrix.php }}
```

Comment:
Rather than a separate step for installing grpc, add here:

```
tools: pecl
extensions: grpc
```

See here for ref: https://github.com/googleapis/google-cloud-php/blob/master/.github/workflows/pubsub-emulator-system-tests.yaml#L24

Reply:
working on it!!